.github/workflows/build-publish-develop.yml

```diff
@@ -55,6 +55,49 @@ jobs:
           dockerhub_username: ${{ secrets.DOCKERHUB_READONLY_USERNAME }}
           dockerhub_password: ${{ secrets.DOCKERHUB_READONLY_PASSWORD }}
           git-commit-sha: ${{ steps.git-ref.outputs.checked-out || github.sha }}
+
+      - name: Setup GAP
+        # Don't run for plugins.
+        if: matrix.image.name == ''
+        uses: smartcontractkit/.github/actions/setup-gap@1bc7ce34fa81fffcb4a6eb0e4e12e59d94d0fc8f # setup-gap@0.2.0
+        with:
+          aws-region: ${{ secrets.AWS_REGION }}
+          aws-role-arn: ${{ secrets.AWS_OIDC_IAM_ROLE_ARN }}
+          api-gateway-host: ${{ secrets.AWS_API_GW_HOST_K8S_SAND }}
+          use-k8s: "true"
+          k8s-cluster-name: ${{ secrets.AWS_EKS_CLUSTER_NAME_SAND }}
+          use-private-ecr-registry: true
+          ecr-private-registry: ${{ secrets.AWS_ACCOUNT_ID_PROD }}
```

Comment:
would we be using the SDLC account?

Reply:
This is going to be on the sandbox CRIBs (using cross-account IAM access) and the ECR images for GAP's local proxy will come from prod 😅